ampersand-multifield-view.js

```diff
@@ -39,19 +33,17 @@ var MultiFieldView = View.extend({
       throw new Error('must pass in a name');
     }
 
-    if (!this.fields) {
-      throw new Error('must have an array of fields');
-    }
-
-    this.value = spec.value || {};
+    this.beforeSubmit = spec.beforeSubmit || this.beforeSubmit;
+    this.fields = spec.fields || this.fields || [];
```

Comment:
-_- I wrote a long comment, and then clicked 'Cancel' by mistake.

The example is when using `AmpersandMultifieldView.extend` to create a new view, say `var MyMFView = MultifieldView.extend({fields: [...]});` and then later overwriting the fields by passing them in during instantiation, which calls initialize: `new MyMFView({fields: [...]});`